syntax = "proto3";

package security;

import "models/util/uuid.proto";
import "models/position/position_filter.proto";

option java_multiple_files = true;
option java_package = "common.request";
option java_outer_classname = "QuerySecurityRequestProtos";

//A request to allow clients to find existing securities.
message QuerySecurityRequestProto {
  string object_class = 1;
  string version = 2;

  //The list of UUIDs to return
  repeated util.UUIDProto uuids = 21;

  //A list of position filters that will filter securities that match.
  position.PositionFilterProto search_security_input = 22;
}


